validator/client/validator.go

```diff
@@ -51,7 +51,8 @@ import (
 // keyFetchPeriod is the frequency that we try to refetch validating keys
 // in case no keys were fetched previously.
 var (
-	keyRefetchPeriod = 30 * time.Second
+	keyRefetchPeriod            = 30 * time.Second
+	MevValidatorRegistrationErr = "MEV validator registration unsuccessful"
```

Comment:
Please prefer the convention of `Err` prefix and make this an actual error. 

```suggestion
	ErrMevValidatorRegistration = errors.New("MEV validator registration unsuccessful")
```